components/tabs/tab-panel-mixin.js

```diff
@@ -7,6 +7,7 @@ export const TabPanelMixin = superclass => class extends superclass {
 		return {
 			/**
 			 * Opt out of default padding/whitespace around the panel
+			 * @type {Boolean}
```

Comment:
Types are generally lower cased, and I believe our `demo-tables` code looks for lower cased values.